students/HarveyHerela/lesson08/Circle.py

```diff
@@ -0,0 +1,99 @@
+from math import pi
+from functools import total_ordering
+
+import random
+import time
+
+@total_ordering
+class Circle:
```

Comment:
class foo(object): is the 'new' way of declaring classes (since python 2.2).

so this should be `class Circle(object)`

Reply:
I take this back :) in Python3 it's always new style class regardless how you define it. My comment only applies to python2.x (greater than 2.2)

Reply:
Since explicit is better than implicit, does that mean that, even though both are correct, it's better style to explicitly show the subclassing of object?